cypress/e2e/20-execution-preview.cy.ts

```diff
@@ -0,0 +1,26 @@
+import { WorkflowPage } from "../pages";
+import { WorkflowExecutionsTab } from "../pages/workflow-executions-tab";
+
+const workflowPage = new WorkflowPage();
+const executionsTab = new WorkflowExecutionsTab();
+
+// Test suite for executions tab
+describe('Current Workflow Executions', () => {
```

Comment:
nitpick - might want to rename the file to `workflow-executions.cy.ts`.. finding the name confusing here with the preview in the iframe

Reply:
Yeah, agree.